### DISCUSSION TITLE
Cloning a Laravel + Oracle 11g Project Requirements | Nazia Shehnaz

### DISCUSSION THREAD

**Discussion by giscus** [Geek-a-Byte/geek-a-byte.github.io#2] [General]
# Cloning a Laravel + Oracle 11g Project Requirements | Nazia Shehnaz

php version : 8.0.3 , OS: Windows, System Specification : 64-bit operating system, x64-based processor

https://geek-a-byte.github.io/posts/Laravel-with-Oracle/

**Comment by wasif6029** [comment]
this is sooo cool!
